tests/integration/test_full_pipeline.py

```
@@ -0,0 +1,410 @@
+    
+    def test_multiple_format_outputs(self, sample_contract_path, tmp_path):
+        """Test generating reports in multiple formats."""
+        orchestrator = ReviewOrchestrator()
+        
+        formats = ["html", "markdown", "json", "text"]
+        
+        for format_type in formats:
+            output_dir = tmp_path / format_type
+            
+            with patch('builtins.input', return_value='y'):
+                result = orchestrator.review_contract(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)